mkt/webapps/management/commands/process_addons.py

```diff
@@ -23,41 +18,11 @@
                                               mkt.STATUS_PUBLIC,
                                               mkt.STATUS_APPROVED],
                                   disabled_by_user=False)]},
-    'add_uuids': {'method': add_uuids,
```

Comment:
Not used anywhere, super old (added in 2012 for https://bugzilla.mozilla.org/show_bug.cgi?id=814131)